leasing/report/invoice/invoicing_review.py

```diff
@@ -336,6 +340,61 @@ def get_incorrect_rent_shares_data(self, service_unit_ids, cursor):
 
         return data
 
+    def get_ongoing_rent_without_rent_shares_data(self, service_unit_ids, cursor):
+        today = datetime.date.today()
+
+        query = """
```

Comment:
I usually name variables like this in UPPERCASE, but not sure what todays agreed convention is

Reply:
flake complains when if I try to name it uppercase